core/model/src/commonMain/kotlin/io/github/droidkaigi/confsched2023/model/DroidKaigi2023Day.kt

```diff
@@ -78,7 +78,7 @@ public enum class DroidKaigi2023Day(
             }
         }
 
-        public fun defaultDyamicThemeDate(): Boolean {
+        public fun defaultDynamicThemeDate(): Boolean {
             return Day1.start < Clock.System.now()
         }
```

Comment:
Oh we don't use this method for this year. Could you remove this?

Reply:
Sure!

Reply:
Changing in https://github.com/DroidKaigi/conference-app-2023/pull/1025/commits/ef613509ed234174f0c90695e05462ec457464ea

Also, Updated the PR title following the actual changes.